addons/godot-next/2d/vector_display_2d.gd

```diff
@@ -0,0 +1,49 @@
+tool
+class_name VectorDisplay2D
+extends Node
+# Displays Vector2 members in the editor via Position2D nodes.
+
+export(String) var variable_name
+export(bool) var relative = true
+
+var _storage: Node2D
+
+
+func _process(delta):
+	if not variable_name:
+		printerr("VectorDisplay2D: Please provide a variable name.")
+		return
+	
+	if not _storage:
+		_storage = Node2D.new()
+		get_tree().get_edited_scene_root().add_child(_storage)
+		return
+	
+	for child in _storage.get_children():
+		child.queue_free()
+	
+	var parent = get_parent()
+	if relative:
+		_storage.transform.origin = parent.global_transform.origin
+	else:
+		_storage.transform.origin = Vector2.ZERO
+	
+	var variable = parent.get(variable_name)
+	if variable == null:
+		printerr("VectorDisplay2D: Variable '" + variable_name + "' not found or invalid on parent node '" + get_parent().get_name() + "'.")
```

Comment:
Also favor `push_error()` over `printerr()` here. Unless I'm mistaken about something.

Reply:
Oh, I wasn't aware that this existed. Sure, I'll change it.